gulp/tasks/dep.js

```diff
@@ -0,0 +1,67 @@
+const fs = require('fs');
+const gulp = require('gulp-help')(require('gulp'));
+const yarn = require('gulp-yarn');
+const main_bower_files = require('main-bower-files');
+const $ = require('gulp-load-plugins')();
+const paths = require('../paths');
+
+gulp.task('yarn', false, () => {
+  gulp
+    .src(['./package.json', './yarn.lock'])
+    .pipe($.plumber())
+    .pipe(yarn());
+});
+
+gulp.task('bower', false, () => {
+  let cmd = 'node_modules/.bin/bower install';
+  if (/^win/.test(process.platform)) {
+    cmd = cmd.replace(/\//g, '\\');
+  }
+  const start_map = [
+    {
+      cmd: cmd,
+      match: /bower.json$/,
+    },
+  ];
+  gulp
+    .src('bower.json')
+    .pipe($.plumber())
+    .pipe($.start(start_map));
+});
```

Comment:
The gulp docs on start are missing, but I suspect re-writing this gulp task so as to not use `start` will fix the problem.

Reply:
can you submit a PR?

Reply:
Sorry, not on this one: I've never properly got to grip with `gulp`.